src/enlyze/client.py

```diff
@@ -265,3 +278,71 @@ def get_timeseries_with_resampling(
             end=end,
             variables=variables_sequence,
         )
+
+    def _get_production_runs(
+        self,
+        *,
+        production_order: Optional[str] = None,
+        product: Optional[str] = None,
+        appliance: Optional[UUID] = None,
+        start: Optional[datetime] = None,
+        end: Optional[datetime] = None,
+    ) -> Iterator[ProductionRun]:
+        """Get production runs from the API."""
+
+        filters = {
+            "production_order": production_order,
+            "product": product,
+            "appliance": appliance,
+            "start": start.isoformat() if start else None,
+            "end": end.isoformat() if end else None,
+        }
+        params = {k: v for k, v in filters.items() if v is not None}
+        return self._production_runs_api_client.get_paginated(
+            "production-runs", ProductionRun, params=params
+        )
+
+    def get_production_runs(
+        self,
+        *,
+        production_order: Optional[str] = None,
+        product: Optional[user_models.Product] = None,
+        appliance: Optional[user_models.Appliance] = None,
+        start: Optional[datetime] = None,
+        end: Optional[datetime] = None,
+    ) -> user_models.ProductionRuns:
+        """Retrieve all production runs.
```

Comment:
Here we should link to the concepts page that explains what a production run is. Moreover, "all" production runs is not accurate because the method provides filtering.

Reply:
The concepts page does not include production runs yet, this is being addressed in #10

Reply:
e9f7495